squad-server/layers/layer.js

```diff
@@ -15,33 +15,33 @@ export default class Layer {
       name: data.lighting,
       classname: data.lightingLevel
     };
-    this.teams = [
-      {
-        faction: data.team1.faction,
-        name: data.team1.teamSetupName,
-        tickets: data.team1.tickets,
-        commander: data.team1.commander,
-        vehicles: (data.team1.vehicles || []).map((vehicle) => ({
+    this.teams = [];
+    this.tanks = [];
+    this.helicopters = [];
```

Comment:
Would it not make more sense for this to be a sub-property of the team? For example:
```js
      this.teams.push({
        faction: data[t].faction,
        name: data[t].teamSetupName,
        tickets: data[t].tickets,
        commander: data[t].commander,
        vehicles: (data[t].vehicles || []).map((vehicle) => ({
          name: vehicle.type,
          classname: vehicle.rawType,
          count: vehicle.count,
          spawnDelay: vehicle.delay,
          respawnDelay: vehicle.respawnTime
        })),
       numberOfHelicopters: // ...
      });
```

Reply:
sure I was working off the old `layers.json` model but this makes more sense